src/renderer/views/Pins.js

```diff
@@ -232,6 +287,49 @@ class Pins extends Component {
                     {rows}
                 </tbody>
             </Table>
+            <h6>Select to pin and help secure the network by backing up videos</h6>
+            <input type='text' placeholder='Enter community ID...' onChange={(event) => {
+                if (event.target.value.match(/\bhive-\d{6}\b/g)) {
+                    this.updateSearchTables(event.target.value)
+                }
+            }} />
+            <Row>
+                {['new', 'trending'].map(type => (
+                    <Col>
+                        <Table striped bordered hover size='sm'>
+                            <thead>
+                            <tr>
+                                <th>{type} videos</th>
+                                <th>Title</th>
+                                <th>Creator</th>
+                                <th>pinned</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            {this.state[`${type}Videos`].map(video => (
```

Comment:
React recommends all elements rendered in an array should have `key=` set to a unique key. In this case that would be the reflink.